docs/dive-in/2024/01/2024-01-08-model-database.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
```

Comment:
We would like to query also all products of categories if not mistaken when making filters for example. You couold add the relation one-to-many from categories to products as one category may have many products.

Reply:
Can you also specify the relation from product to categories?

Reply:
You can check the docuementation here : https://mermaid.js.org/syntax/classDiagram.html#cardinality-multiplicity-on-relations